projects/observability/src/shared/components/timeline-card-list/container/timeline-card-container.component.ts

```diff
@@ -9,4 +9,7 @@ import { ContentHolder, CONTENT_HOLDER_TEMPLATE } from '@hypertrace/components';
 export class TimelineCardContainerComponent extends ContentHolder {
   @Input()
   public timestamp!: number | Date;
+
+  @Input()
+  public similarToPrevious?: boolean;
```

Comment:
This API strikes me as pretty strange - components don't generally know about their siblings.

Reply:
explained below